spec/frame_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+describe Frame do
+  let(:frame) { Frame.new(1) }
+
+  describe '#run' do
+
+    it 'correctly records a strike' do
+      $stdin = UserInput.input([10])
+      frame.run
+      expect(frame.strike?).to eq(true)
```

Comment:
I believe that for regular RSpec best convention is to try and keep tests to one expect statement. Consider the have_attributes matcher to eliminate multiple expectations in one test, but still test the attributes are set correctly.

https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/have-attributes-matcher